### PR TITLE
Named Bindings Support

### PR DESCRIPTION
**Pull request by autodidaddict**
Changes to the codec to support the notion of named bindings. Additional refactoring was done at the same time, including removing unnecessary wrappers around the broker message, etc. Further annotations were added to aid in msgpack serialization to ensure that we have round-trip compatibility with the AssemblyScript code.

### CODE REVIEW

**Comment by autodidaddict** [comment]
Another subtle (but breaking) change is that the `dispatch` function used by capability providers to invoke actors now requires an actor explicitly. In the previous versions, we've been using string concatenation with a separator char (`!`) to munge the operator and the actor... which was :hankey:

File: src/http.rs [hunk 1]
```diff
@@ -1,33 +1,83 @@
 //! # HTTP server capability data structures
 //!
-//! This module contains data types for the `wascc:http_server` and `wascc:httpclient` capabilities.
+//! This module contains data types for the `wascc:http_server` and 1wascc:httpclient` capabilities
```
Comment (brooksmtownsend) [inline src/http.rs:3]: Looks like you meant to have a ` here

File: src/keyvalue.rs [hunk 2]
```diff
@@ -20,80 +22,105 @@ pub const OP_KEY_EXISTS: &str = "KeyExists";
 
 /// A request to get a single value from the K/V store
 #[derive(Debug, PartialEq, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
```
Comment (brooksmtownsend) [inline src/keyvalue.rs:25]: Just curious as I keep seeing this, why rename to camelCase?

File: src/http.rs (hunk 1 above)
Comment (autodidaddict) [reply src/http.rs:3]: TYPO? INEXCUSABLE.

File: src/keyvalue.rs (hunk 2 above)
Comment (autodidaddict) [reply src/keyvalue.rs:25]: the other languages we want to support all default to camelCase naming convention, including the schema language that we want to use for generated code. So we're just defaulting the "wire format" of the serialization to camelCase. Unlike protobufs, the field names in msgpack transfer over the wire.

**Pull request merged**